### ISSUE TITLE
English Readme

### ISSUE THREAD

**Issue by MatthewForrester** [TVTower/TVTDatabaseEditor#13]
I have started to translate the readme into English in [this branch](https://github.com/MatthewForrester/TVTDatabaseEditor/tree/english-readme). 

Is the editor usable now or is it still at an early stage of development?

**Comment by nittka** [comment]
It is not yet the time for translating the readme into English. The editor is in a very early development stage. Basically, I am trying to do three things simultaneously at the moment:
* documenting the database format (https://github.com/TVTower/Documentation/tree/databaseDocumentation/database_de) - Branch in the (temporary?) documentation repository
* clean up the existing database files (redundant definition possibilities, ...)  (https://github.com/TVTower/TVTower/pull/346)
* implement an Xtext based text editor for the cleaned up format

Progress is naturally slow (and time limited) and there is much work left. If you are familiar with Eclipse, you might already use the editor, but it has virtually no interesting features, yet. The readme was written for those who want to take a look at what might be coming. But you have to build the editor, yourself.
You can download a snapshot Eclipse update site from the actions workflows (https://github.com/TVTower/TVTDatabaseEditor/actions) of this repository, so you are not forced to build the plugin yourself.

To summarize, translating the readme is not really fruitful, as it will be a lot different, once the editor is in a more presentable state. (Note that the same still applies to the database documentation. It is currently a rough draft and working on it consistently in two languages at the same time is too much effort... Get the German version stable then translate it and do updates simultaneously, then.)

**Comment by nittka** [comment]
I want to apologize if my last reply was offending - it was not intended.

I plan to publish a first (alpha-) version of the editor in the near future. The first cleanup of the database (main repository) and a first documentation version for the database format (documentation repository) have been pushed to master. When the last cleanup for the editor is done, I will also rewrite the readme.

**Comment by MatthewForrester** [comment]
> I want to apologize if my last reply was offending - it was not intended.

Not at all. Apologies for the slow reply; I have just been playing other games, that's all.

I just thought I would translate the readme as I found it quite relaxing translating the manual. I can easily come back to this readme at some point in the future.

**Comment by nittka** [comment]
I have created the English readme. An initial german documentation for the editor has been added to the documentation repository. The English translation will shortly follow.
